src/test/resources/harness-config.yml

```diff
@@ -308,3 +308,11 @@ databasesUnderTest:
     url: jdbc:firebirdsql://localhost:3050/lbcat
     username: lbuser
     password: LiquibasePass1
+
+  - name: informix
+    prefix: docker
+    version: 14
+    url: jdbc:informix-sqli://localhost:9088/sysmaster:INFORMIXSERVER=informix;
```

Comment:
need to change sysmaster to some testdb
but was not able to make entrypoint work at the moment 
    #volumes:
    #- "./informix-init.sh:/docker-entrypoint-initdb.d/informix-init.sh"
    #- "./informix-init.sql:/docker-entrypoint-initdb.d/informix-init.sql"
    #entrypoint: /bin/bash /docker-entrypoint-initdb.d/informix-init.sh

Reply:
https://github.com/informix/informix-dockerhub-readme/blob/master/14.10.FC1/informix-developer-database.md
this might work
```
User Supplied sch_init_xxxxx.sql You can specify an sch_init_xxxxx.sql file to be used by placing the file in hosts directory that is mounted to the config volume. In the example above it would be /home1/setupvol. This file is used during startup to perform additional initialization options. ex. creation of dbspaces, databases, etc. Use the -e option to specify the file name.
      -e INIT_FILE=my_init.sql
```